src/synopsys-action/artifacts.ts

```diff
@@ -51,11 +65,20 @@ export function getFiles(dir: string, allFiles: string[]): string[] {
   return allFiles
 }
 
-export async function uploadSarifReportAsArtifact(defaultSarifReportDirectory: string, userSarifFilePath: string, artifactName: string): Promise<UploadResponse> {
-  const artifactClient = artifact.create()
+export async function uploadSarifReportAsArtifact(defaultSarifReportDirectory: string, userSarifFilePath: string, artifactName: string): Promise<UploadArtifactResponse> {
+  const githubServerUrl = process.env[constants.GITHUB_ENVIRONMENT_VARIABLES.GITHUB_SERVER_URL] || ''
+  const isGitHubCloud: boolean = githubServerUrl === constants.GITHUB_CLOUD_URL
```

Comment:
As this check is happening at multiple places, we can have this check at one place (may be in utility) to reduce redundancy.

Reply:
Resolved